pkg/platform/platform.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2024 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package platform defines interfaces for interacting with code review
+// platforms.
+package platform
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/abcxyz/guardian/pkg/github"
+)
+
+const (
+	TypeLocal  string = "local"
+	TypeGitHub string = "github"
```

Comment:
These exist in the flags/global.go file now too, we should pick a single place (probably here) and refactor.

Reply:
Moved to platform package